### ISSUE TITLE
[GlobalStep] iOS -  Tapping twice in an empty area while in the Block Editor causes the keyboard to be displayed and dismissed constantly.

### ISSUE THREAD

**Issue by wptester9845** [wordpress-mobile/gutenberg-mobile#2067]
### Description
While in the Block Editor, tapping twice in an empty area causes two Paragraph blocks to be created, however, the selected block will then constantly switch between these blocks while the keyboard is displayed and and dismissed.
This issue was only seen to occur on iOS.

### Reproduction Rate
4/4 100%

### Expected behavior
One of the blocks should remain selected and the keyboard displayed or hidden.

### Actual behavior
The selected block constantly switches while the keyboard is displayed and hidden.

### Steps to reproduce the behavior
1. Install WordPress 14.5
2. Log in to a valid account.
3. Press the “Add new” button.
4. Select the “Blog Post” option.
5. Double tap the empty area in the block editor.
##### Tested on the following
iPhone 5S (11.4)

#### Please see the attached video for more information
[BlockFlicking.zip](https://github.com/wordpress-mobile/gutenberg-mobile/files/4387785/BlockFlicking.zip)

### Submitted by:
Luis Pimenta

**Comment by designsimply** [comment]
Tested and confirmed. In my testing, I noticed that the problem with the keyboard didn't start happening until the content started to overlap with the keyboard. I also found that minimizing and unminimizing the app didn't make the problem go away—I had to close the app completely and re-open it to get the keyboard problem to stop. ([55s](https://cloudup.com/cOPFg-AglvO))

<sup>Tested with WPiOS 14.5.0.1 TestFlight beta on iPhone 6S iOS 13.3.1.</sup>
